fireant/slicer/queries.py

```diff
@@ -104,12 +104,16 @@ def query_data(self, database, table, joins=None,
             A pd.DataFrame indexed by the provided dimensions paramaters containing columns for each metrics parameter.
         """
         query = self._build_data_query(table, joins, metrics, dimensions, dfilters, mfilters, references, rollup)
-
         query_string = str(query)
-        logger.info("Executing query:\n----START----\n{query}\n-----END-----".format(query=query_string))
 
+        start_time = time.time()
         dataframe = database.fetch_dataframe(query_string)
 
+        query_logger.debug('[duration: {duration} seconds]: {query}'.format(
```

Comment:
can you make this info please? it's not for debugging, it's for logging which queries get executed